task.js

```diff
@@ -148,21 +155,9 @@ const tasks = {
 
     fixSourcePaths(coverage)
 
-    const previousCoverage = existsSync(nycFilename)
-      ? JSON.parse(readFileSync(nycFilename, 'utf8'))
-      : {}
-
-    // previous code coverage object might have placeholder entries
-    // for files that we have not seen yet,
-    // but the user expects to include in the coverage report
-    // the merge function messes up, so we should remove any placeholder entries
-    // and re-insert them again when creating the report
-    removePlaceholders(previousCoverage)
+    console.log(coverageMap.toJSON())
```

Comment:
```suggestion
```

Reply:
Why was the removePlaceholders logic removed?

Reply:
@koenvangeert Sorry for the long response time. Are you able to respond to Emily's review?

Reply:
@jennifer-shehane Sorry for my long response time 🙇 . I fixed the comments, can I do something do push this along?

Reply:
We'll take another look. Thanks!